onedal/decomposition/pca.py

```diff
@@ -15,30 +15,75 @@
 # ==============================================================================
 
 import numpy as np
+from sklearn.decomposition._pca import _infer_dimension
+from sklearn.utils.extmath import stable_cumsum
 
 from daal4py.sklearn._utils import sklearn_check_version
 
 from ..common._base import BaseEstimator
-from ..common.hyperparameters import get_hyperparameters
+from ..common._policy import _get_policy
```

Comment:
please do not use _get_policy directly. This estimator inherited from BaseEstimator, that already have this method